src/components/felles/dobbelinfo.tsx

```diff
@@ -0,0 +1,24 @@
+import '../info.css';
+import { StringOrNothing } from '../../utils/felles-typer';
+
+interface Props {
+    header: string;
+    values?: StringOrNothing[];
+    children?: React.ReactNode;
+}
+export function DobbeltInformasjon(props: Props) {
```

Comment:
Denne komponenten er ment for å gjenbrukes / starteksempel til de andre boksene som skal lages.